packages/shared/src/mock/metadata.ts

```diff
@@ -61,7 +62,7 @@ export const mockMetadata = makeNxGroup(mockFilepath, 'NXroot', {
         makeNxGroup('nx_process', 'NXprocess', {
           children: [
             makeNxDataGroup('nx_data', {
-              signal: makeNxDataset('twoD', intType, [20, 41]),
+              signal: makeDataset('twoD', intType, [20, 41]),
```

Comment:
Just a simplification here -- we don't pass `units` and the like, so we don't need to use `makeNxDataset`.